m

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+cd "$(dirname "$0")"
+
+# Ensure submodules are cloned and up to date.
+git submodule update --init
+
+container_name=code-server-dev
+
+# Docker build is very verbose even when everything is cached.
+echo "--- Building $container_name"
+docker build -t $container_name -f ./ci/dev.Dockerfile . > /dev/null
+
+set +e
+container_id=$(docker container inspect --format="{{.Id}}" $container_name 2>/dev/null)
+
+if [ $? -eq "0" ]; then
+    echo "--- Killing $container_name"
+    docker rm -f $container_name 2>/dev/null
+fi
+set -e
+
+echo "--- Spawning $container_name"
```

Comment:
if the container is already running, we should just reuse it.

Reply:
Actually I guess that wouldn't be very useful since you can just use the VS Code terminal.

Reply:
Having to use the code-server terminal would not be optimal though considering it's constantly reloading, losing its history, etc.